src/runtime/hexagon_remote/c11_stubs.cpp

```diff
@@ -0,0 +1,23 @@
+extern "C" {
+
+// Hexagon-tools 8.0.06 and later are dependent on 2 additional symbols:
+//    __cxa_finalize and __cxa_atexit
+// for backward compatability with hexagon_remote runtimes, we are
```

Comment:
It's for compatibility with older toolchains/libraries, right? This *is* the runtime, so compatibility with the runtime doesn't quite make sense.